package.json

```diff
@@ -21,8 +21,8 @@
     "precompile": "npm run lint",
     "prepublish": "npm run build:release",
     "preversion": "npm run test && git add README.md CHANGELOG.md",
-    "start": "npm run clean && npm run build && concurrently \"onchange 'src/**/*.ts' -- npm run build\" \"karma start\"",
-    "test": "npm run build:release && karma start --single-run=true --browsers Firefox"
+    "start": "karma start",
+    "test": "karma start --single-run=true --browsers Firefox"
```

Comment:
must feel good. :smile:

Reply:
yeah, _but_ coverage now seems to point to the wrong lines in the TS files.